src/centos/7/mlnet/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7
```

Comment:
@MichaelSimons - can you confirm if this is correct?
Also, how do I test this?

Reply:
You can test via `.\build.ps1 -DockerfilePath "centos/7*"`
Yes these should be part of the build.

Reply:
You will need to add these to the manifest.json at the root of the repo in order for them to get built.

Reply:
See [How to modify or create a new image](https://github.com/dotnet/dotnet-buildtools-prereqs-docker#how-to-modify-or-create-a-new-image) for details on this.